Android/res/drawable/geotag_button.xml

```diff
@@ -0,0 +1,21 @@
+<?xml version="1.0" encoding="utf-8"?>
+<selector xmlns:android="http://schemas.android.com/apk/res/android">
+    <item android:state_pressed="true">
+        <shape android:shape="rectangle">
```

Comment:
@chaviw you're repeating the same shape for all three states. Instead you can define the shape once, and set a selector for the `solid` color attribute.